cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/NamespaceHttpHandler.java

```diff
@@ -142,15 +134,11 @@ public void create(HttpRequest request, HttpResponder responder, @PathParam("nam
       .build();
 
     try {
-      NamespaceMeta existing = store.createNamespace(builder.build());
-      // make the API idempotent, but send appropriate response
-      String response;
-      if (existing == null) {
-        response = String.format("Namespace '%s' created successfully.", namespaceId);
-      } else {
-        response = String.format("Namespace '%s' already exists.", namespaceId);
-      }
-      responder.sendString(HttpResponseStatus.OK, response);
+      namespaceService.createNamespace(builder.build());
+      responder.sendString(HttpResponseStatus.CREATED,
```

Comment:
Reverting this change since we decided to keep it as 200 per [this cdap-dev thread](https://groups.google.com/forum/?hl=en#!topic/cdap-dev/WyMKuPjeqmE)